sympy/core/tests/test_relational.py

```diff
@@ -183,18 +184,10 @@ def test_doit():
 def test_new_relational():
     x = Symbol('x')
 
-    assert Eq(x) == Relational(x, 0)       # None ==> Equality
-    assert Eq(x) == Relational(x, 0, '==')
-    assert Eq(x) == Relational(x, 0, 'eq')
-    assert Eq(x) == Equality(x, 0)
```

Comment:
I think these tests should be changed to `Eq(x, 0)` rather than removed

Reply:
They are handled below.

Reply:
I see what you mean. I'll mark as resolved.